service/src/intTest/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcStructuredComponentTest.java

```diff
@@ -75,7 +76,9 @@ public void When_NewStructuredTask_Expect_DatabaseUpdatedAndAddedToObjectStore()
         ehrExtractStatusRepository.save(ehrExtractStatus);
 
         GetGpcStructuredTaskDefinition structuredTaskDefinition = buildValidStructuredTask(ehrExtractStatus);
-        getGpcStructuredTaskExecutor.execute(structuredTaskDefinition);
+        // temporarily ignore the test while GPC data is invalid
```

Comment:
Can we reference the ticket we've opened to track/fix this? Same for comments below